snake.js

```diff
@@ -32,20 +32,42 @@ class GameBoard {
     }
 }
 
-class SnakeGame{
+class SnakeGame {
     constructor() {
         this.game = document.createElement('canvas');
         this.game.width = BOARD_X * (MODULE_SIZE + 1) - 1;
         this.game.height = BOARD_Y * (MODULE_SIZE + 1) - 1;
         this.board = new GameBoard(this.game, 'lightgray');
+        this.wall = new Wall(this.game);
         this.render();
+
     }
-    build() {};
+    build() { };
     render() {
         this.board.draw();
+        this.wall.draw(5,6);
     };
 }
 
-const snake = new SnakeGame();
+class Wall {
+    constructor(gameCanvas) {
+        this.game = gameCanvas.getContext('2d');
+        this.model = document.createElement('canvas');
+        this.ctx = this.model.getContext('2d');
+        this.model.width = MODULE_SIZE;
+        this.model.height = MODULE_SIZE;
+        this.build();
+    }
+    build() {
+        this.ctx.fillStyle = "red";
+        this.ctx.fillRect(0, 0, MODULE_SIZE, MODULE_SIZE);
```

Comment:
We have to agree on some wall "looks" - filled rectangle doesn't look appealing at all.